### PR TITLE
Tnw

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
The Next Web Website Clone.
Done with HTML, CSS.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Status: changes required :gear: 
Impressive work TEAM.**
You have created a perfect clone of the original page. Layout and responsiveness are very good. But there are a couple of improvements to be made to get approved. I am confident that it will not take that long to change. Please make the following changes.
**[NOTE] optional changes are not mandatory but nice to have.**

[Required] Remove the vertical lines on left and right side of the hamburger menu.
Screenshot from 2020-06-21 21-50-15

[Required] Let's use the same colors as on the original page for the footer backgrounds and texts below article titles.
### changes for medium screen
[Required] Let's give more width to the secondary articles.
your layout :point_down: 
Screenshot from 2020-06-21 22-06-32

Original layout :point_down: 
Screenshot from 2020-06-21 22-06-15

File: main.css [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/apps.png);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.gear-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/gear.png);
+  background-position: center center;
+  background-size: cover;
```
Comment (sumancrest0001) [inline main.css:378]: [Required] Even though the stylings of elements are good. there are a lot of repeated codes that are not necessary. Let's follow the DRY principle and optimize the CSS file. [This article](https://speckyboy.com/good-bad-css-practices/) might be helpful.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+}
+
+.grid-address a:hover,
+.news p a:hover {
+  color: black;
+}
+
+.list-grid h2 a:hover {
+  color: black;
+}
```
Comment (sumancrest0001) [inline main.css]: [Required] Even though the stylings of elements are good. there are a lot of repeated codes that are not necessary. Let's follow the DRY principle and optimize the CSS file. [This article](https://speckyboy.com/good-bad-css-practices/) might be helpful.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the requested changes and fill up another review form.
If you have any questions regarding the review, DM me on slack Suman Shrestha
Happy coding !!!!**

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @MahmoudBakr23  and @Abidoyinsola1! 👋👋  
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] I see that you have removed the vertical lines as was requested by the previous TSE 👍 However, the layout navigation bar on the medium screen now appears to have broken. Please fix this. 👇 
Screen Shot 2020-06-21 at 10 26 28 PM

- [ ] Please see my comments for additional changes. Please use my comments as a guide and search for more instances of repeated CSS rules. 


****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: main.css [hunk 3]
```diff
@@ -0,0 +1,769 @@
...
+  color: #999;
+}
+
+.list-1 li {
+  margin-top: 0.2em;
+}
+
+.list-2 li {
+  margin: 0 0.2em;
+}
```
Comment (BrittanyBlake) [inline main.css]: As requested by the previous TSE, please use the DRY principle to optimize your CSS file

File: main.css [hunk 4]
```diff
@@ -0,0 +1,769 @@
...
+    justify-content: space-between;
+  }
+
+  .bar-two .list-2 {
+    display: none;
+  }
+
+  .bar-one {
+    display: none;
+  }
```
Comment (BrittanyBlake) [inline main.css]: As requested by the previous TSE, please use the DRY principle to optimize your CSS file

File: main.css [hunk 5]
```diff
@@ -0,0 +1,769 @@
...
+
+  .bar-two .list-1,
+  .bar-two .list-2 {
+    display: none;
+  }
```
Comment (BrittanyBlake) [inline main.css]: As requested by the previous TSE, please use the DRY principle to optimize your CSS file

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
**HELLO** @MahmoudBakr23  **VERY IMPRESSIVE WORK SO FAR** :+1: 
You have done such a great job implementing the changes required by the previous TSE :+1: :clap: 

However, there are a few changes that require to be made. KIndly work on the following and submit your project for another review :smile: 

- [ ]  There are still some responsiveness issues in your menu bar Kindly fix this issues.

menu responsive

- [ ] I have noticed that there are some CSS issues in your styles. You can use this [tool](https://jigsaw.w3.org/css-validator/validator) to double-check your CSS
css errors

kindly submit your project once you fix the issues above :smile: 

If you have any question concerning the review kindly contact me via slack on @peterrobert

### HAPPY CODING :computer:

**Review by elmejdki — APPROVED** [review]
# Wooow, Great job guys :clap: your clone looks amazing :tophat:
### status: approved

you can merge now to master your project was approved.

### Happy Coding. :smiley:

**Pull request merged**